docs/source/scale/sagemaker.rst

```diff
@@ -196,7 +197,8 @@ Users can use the following command to launch a GraphStorm Link Prediction infer
 
 .. note::
 
-    Diffferent from the training command's argument, in the inference command, the value of the ``--model-artifact-s3`` argument needs to be path to a saved model. By default, it is stored under an S3 path with specific training epoch or epoch plus iteration number, e.g., ``s3://models/epoch-0-iter-999``, where the trained model artifacts were saved.
+    * Different from the training command's argument, in the inference command, the value of the ``--model-artifact-s3`` argument needs to be path to a saved model. By default, it is stored under an S3 path with specific training epoch or epoch plus iteration number, e.g., ``s3://models/epoch-0-iter-999``, where the trained model artifacts were saved.
+    * If ``--raw-node-mappings-s3`` is not provided, it will be default to the ``{graph-data-s3}/raw_id_mappings``. The expected graph mappings files should be ``node_mapping.pt``, ``edge_mapping.pt`` and parquet files under ``raw_id_mappings``. They record the mapping between original node and edge ids in the raw data files and the ids of nodes and edges in the Graph Node ID space.
```

Comment:
Shouldn't an rst list be 

```
- item1
- item2
```

Also be aware, when breaking up lines in one list item in rst ensure the lines per item start at the _exactly_ same point e.g.

```
- item1 with
  very long text.
- item2
```

something like

```
- item1 with
    very long text.
- item2
```

will lead to wrong formatting.

Reply:
I think it could be both. But at least we should keep the same format in this doc. Refer to the https://github.com/awslabs/graphstorm/blob/90b4233687ff3662ba51dfdf211da3c694dd16eb/docs/source/scale/sagemaker.rst?plain=1#L27. We use * for the note::.